Library/Formula/scirius.rb

```diff
@@ -0,0 +1,49 @@
+class Scirius < Formula
+  desc "Rules manager for Suricata IDPS"
+  homepage "https://github.com/StamusNetworks/scirius"
+  url "https://github.com/StamusNetworks/scirius"
```

Comment:
Hi I'm not sure if such links exist... I've installed it using:
    $ git clone https://github.com/StamusNetworks/scirius /usr/local/var/www/scirius
https://github.com/Homebrew/homebrew/issues/41499
what should I do?